src/app/_variables.scss

```diff
@@ -1,5 +1,8 @@
 $yellow: #FFC100;
-$blue: #0019FF;
+$blue_new: #00B3F9;
+$blue_quite_pure: #2140ff;
+$blue_pure: #0019FF;
+$blue: $blue_new;
```

Comment:
$blue = $blue_new ? why not just `$blue` and regarding the actual color should be more `$blue_light`

Reply:
@dsmrs I just did that for dev purposes to keep a list of all the blues we like
`$blue_new, $blue_quite_pure, $blue_pure` and simply update $blue quickly